pallets/rc-migrator/src/accounts.md

```diff
@@ -114,3 +114,92 @@ Source: https://github.com/paritytech/polkadot-sdk/blob/ace62f120fbc9ec617d6bab0
+native token (DOT/KSM), and through this account balance, track all "exported" DOT/KSM. Summing
+that with the total balance of all other local accounts provides the token's *total issuance*.  
+On top of that, the checking account is also used to enforce that the amount of DOT/KSM teleported
+"back in" cannot surpass the amount teleported "out".
+
+During AHM, we move all of the above responsibilities from the Relay Chain to Asset Hub. AH will be
+the source of truth for DOT/KSM *total issuance*.
+
+#### Migration Design Assumptions
+
+1. AHM has no implications for the other System Chains' checking accounts - only Relay and AH.
+2. Migration of checking account balance falls under base case of generic account migration: no
+   special handling when "moving funds" of this account: balance of checking account on Relay will
+   simply be migrated over to the same checking account on Asset Hub using same logic and part of
+   the same process as all other accounts.
+3. Not **all** DOT/KSM will be moved from Relay Chain to AH. Some of it will stay put for various
+   reasons covered throughout this doc.
+4. The new balance of checking account on AH will be adjusted in a dedicated migration step to
+   properly account for the "exported" tokens. The logic for this calculation is described further
+   down.
+5. To avoid having to coordinate/track DOT/KSM teleports across all System Chains during AHM,
+   DOT/KSM teleports shall be disabled for Relay Chain and Asset Hub during accounts migration.
+
+| DOT Teleports (in or out) | **Relay** | **AH** |
+|----------|-----|--------|
+| _Before_ | Yes | Yes |
+| _During_ | No  | No  |
+| _After_  | Yes | Yes |
+
+
+The runtime configurations of Relay and AH need to change in terms of how they use the checking
+account before, during and after the migration.
+
+|     DOT Teleports tracking in Checking Account    | **Relay** | **AH** |
+|----------|-----------|--------|
+| _Before_ |     Yes, MintLocal       |   No Checking     |
+| _During_ |     No Checking       |   No Checking     |
+| _After_  |     No Checking      |   Yes, MintLocal     |
+
+#### Tracking Total Issuance Post-Migration
+
+Pre-migration RC checking account tracks total DOT/KSM that "left" RC and is currently on some other
+system chain. The DOT/KSM in various accounts on AH is also tracked in this same RC checking account.
+
+Post-migration, we want the tracking to move to AH. So AH checking account will track total DOT/KSM
+currrently living on RC or other system chains.
+
+The **important invariant** here is that the DOT/KSM **total issuance** reported by RC pre-migration
+matches the total issuance reported by AH post-migration.
+
+To achieve this, we implement the following arithmetic algorithm:
+
+After all accounts (including checking account) are migrated from RC to AH:
+
+```
+	ah_checking_intermediate = ah_check_before + rc_check_before
```

Comment:
looks good to me, but I doubt someone without context will understand from just reading this. even some var names not clear, if you do not know it, like `ah_check_before`, when `ah_checking_intermediate` has full `checking` in its name.

Reply:
I added a drawing better explaining it